### ISSUE TITLE
empty output folder

### ISSUE THREAD

**Issue by kookykriss** [boy1dr/SpleeterGui#40]
Hello,
I am on windows 10 with version 2.6 of Spleeter.
At first everything was correct and for a while I have nothing in the output folder. Spleeter normally works but nothing on file. I have reinstalled Spleeter 2.6 several times but nothing can be done.
Thank you for your help
Kooky

**Comment by boy1dr** [comment]
Hi Kooky
There is an FAQ page here https://makenweb.com/spleeter_help.php maybe that could be of some help.
Otherwise... when SpleeterGUI runs it should write some text below the progress bar, can you paste that in here?
That should tell me whats going wrong for you.

**Comment by kookykriss** [comment]
Hello Chris,
as you probably see in the picture i sent everything seems to work properly with scripts and progress bar.
However there is no file at all at the end. I thought that this problem might come from my pc but is quite impossible
because it worked at start. Thank you for your help
Kooky
    Le vendredi 24 juillet 2020 à 12:04:27 UTC+2, Chris Mitchell <notifications@github.com> a écrit :  
 
 


Hi Kooky
There is an FAQ page here https://makenweb.com/spleeter_help.php maybe that could be of some help.
Otherwise... when SpleeterGUI runs it should write some text below the progress bar, can you paste that in here?
That should tell me whats going wrong for you.

**Comment by boy1dr** [comment]
Sorry i can't see any picture.
But if there's no error message and it seems to work then i can only guess that there's a file path issue.
Are you using another language other than English? or does the song name or path have any non English characters in it?

**Comment by kookykriss** [comment]
In fact I created a folder called Spleeter on my desktop in which the subfolders go while working and at the end I have the subfolder in my folder but empty. As far as the names of the songs are concerned, they are French characters, finally French titles so I don't see
    Le vendredi 24 juillet 2020 à 13:56:41 UTC+2, Chris Mitchell <notifications@github.com> a écrit :  
 
 


Sorry i can't see any picture.
But if there's no error message and it seems to work then i can only guess that there's a file path issue.
Are you using another language other than English? or does the song name or path have any non English characters in it?

**Comment by boy1dr** [comment]
I have tested a song named "Le vendredi 24 juillet 2020 à.mp3" and it appears to work fine for me.
My concern is that my handling of strings is not coping with french language.
if you could rename your song to "c:\song.mp3" and set SpleeterGUI output to "c:\" and see if it works that would help me a lot.

Also what is the full name of song?
Also what is the full output path?

**Comment by kookykriss** [comment]
Problem solved it was a problem of out folder. Thank you very  much for your help

Envoyé depuis Yahoo Mail pour Android 
 
  Le ven., juil. 24, 2020 à 14:33, Chris Mitchell<notifications@github.com> a écrit :   


I have tested a song named "Le vendredi 24 juillet 2020 à.mp3" and it appears to work fine for me.
My concern is that my handling of strings is not coping with french language.
if you could rename your song to "c:\song.mp3" and set SpleeterGUI output to "c:" and see if it works that would help me a lot.

Also what is the full name of song?
Also what is the full output path?

**Comment by boy1dr** [comment]
great to hear :)
was it a problem with my software ?

**Comment by kookykriss** [comment]
No problem With Spleeter but i  still have a question for you..Do you think one day spleeter will be  improved to get the possibility of keeping chorus of the song without damage instruments at all. In fact will we get a real play-back in the future?

Envoyé depuis Yahoo Mail pour Android 
 
  Le ven., juil. 24, 2020 à 14:50, Chris Mitchell<notifications@github.com> a écrit :   


great to hear :)
was it a problem with my software ?

**Comment by boy1dr** [comment]
i think effects like echo/reverb/chorus make it hard for spleeter to work.
it is possible for it to work better with a larger training database especially if the instruments you want to extract are not in the training set.

**Comment by kookykriss** [comment]
Hello Chris,
Thank you very much for your answer .
In fact my question was: will Spleeter still make progress, improve in the future. One day will we be able to know the happiness of creating a real play-back? My dream would be to keep the instruments and the chorus and to remove only the lead vocal of the singer as if it were a real orchestral playback. I know it's difficult but I never thought we would have come to this before. So I think everything is possible but as I don't want to bother you chris, you will answer me when you can and I wish you a pleasant Sunday
    Le vendredi 24 juillet 2020 à 12:04:27 UTC+2, Chris Mitchell <notifications@github.com> a écrit :  
 
 


Hi Kooky
There is an FAQ page here https://makenweb.com/spleeter_help.php maybe that could be of some help.
Otherwise... when SpleeterGUI runs it should write some text below the progress bar, can you paste that in here?
That should tell me whats going wrong for you.

**Comment by kookykriss** [comment]
Hello Chris,
Thank you very much for your answer .
In fact my question was: will Spleeter still make progress, improve in the future. One day will we be able to know the happiness of creating a real play-back? My dream would be to keep the instruments and the chorus and to remove only the lead vocal of the singer as if it were a real orchestral playback. I know it's difficult but I never thought we would have come to this before. So I think everything is possible but as I don't want to bother you chris, you will answer me when you can and I wish you a pleasant Sunday

Envoyé depuis Yahoo Mail pour Android 
 
  Le dim., juil. 26, 2020 à 2:38, Chris Mitchell<notifications@github.com> a écrit :   


i think effects like echo/reverb/chorus make it hard for spleeter to work.
it is possible for it to work better with a larger training database especially if the instruments you want to extract are not in the training set.

**Comment by kookykriss** [comment]
Hello Chris,
As I have not had an answer to my questions, I allow myself
to ask again if you think Spleeter will evolve further in the future? Can we hope one day to obtain a real orchestral playback while keeping the chorus and without damaging the instruments?
Is there any research done in this direction? thank you and sorry for bothering you
    Le dimanche 26 juillet 2020 à 02:38:39 UTC+2, Chris Mitchell <notifications@github.com> a écrit :  
 
 


i think effects like echo/reverb/chorus make it hard for spleeter to work.
it is possible for it to work better with a larger training database especially if the instruments you want to extract are not in the training set.

**Comment by boy1dr** [comment]
Hi Kooky
Yes i think spleeter will get better...but maybe not to the level you are hoping for.
Deezer made spleeter for a different purpose to what we want to use it for and for them it is just about perfect.
I think in the future software like spleeter will be the first of several operations.
1. (spleeter) separate the music in to individual instruments.
2. identify notes and tonalities.
3. replicate and reproduce the instruments
4. add DSP effects an mix audio levels to fill out the audio spectrum.

This isn't too different from the early days of the "codec". what we here in our calls on our cellphones is a close approximation of the audio recorded on the other end, not a replaying of audio samples. for us it's good enough, sometimes we sound like robots.
As this sort of technology gets better we will start to hear perfection but i think we are a few years off that yet.
especially for real-time processing...that may never happen. 
perhaps with a few seconds delay and fast processing hardware we will one day have the equivalent of audio augmented reality.

**Comment by kookykriss** [comment]
thank you christ for all these details and for your interest in my questions

    Le mardi 28 juillet 2020 à 13:58:24 UTC+2, Chris Mitchell <notifications@github.com> a écrit :  
 
 


Hi Kooky
Yes i think spleeter will get better...but maybe not to the level you are hoping for.
Deezer made spleeter for a different purpose to what we want to use it for and for them it is just about perfect.
I think in the future software like spleeter will be the first of several operations.
   
   - (spleeter) separate the music in to individual instruments.
   - identify notes and tonalities.
   - replicate and reproduce the instruments
   - add DSP effects an mix audio levels to fill out the audio spectrum.

This isn't too different from the early days of the "codec". what we here in our calls on our cellphones is a close approximation of the audio recorded on the other end, not a replaying of audio samples. for us it's good enough, sometimes we sound like robots.
As this sort of technology gets better we will start to hear perfection but i think we are a few years off that yet.
especially for real-time processing...that may never happen.
perhaps with a few seconds delay and fast processing hardware we will one day have the equivalent of audio augmented reality.
